packages/suite/src/reducers/suite/suiteReducer.ts

```diff
@@ -245,4 +245,9 @@ export const selectTorState = createSelector(
     }),
 );
 
+export const selectDebug = createSelector(
```

Comment:
this is somewhat wrong, should be just 

```javascript
export const selectDebug = (state: SuiteRootState) => state.suite.settings.debug;
```

see the explanation here:
https://redux.js.org/usage/deriving-data-selectors
![image](https://user-images.githubusercontent.com/3729633/207831317-25798820-8f1d-44f4-9328-aa46e487deec.png)


I will rewrite it in the https://github.com/trezor/trezor-suite/pull/7189

Reply:
https://github.com/trezor/trezor-suite/pull/7228